README.md

```diff
@@ -122,7 +123,8 @@ List of links to the various checkers out there on the web for sites, domains, s
 | URL | Description | 💳 |
 | --- | ----------- | -- |
 | https://www.checkbot.io/ | Browser extension that checks multiple pages at once for web security best practices. Finds HTTPS mixed content issues, insecure password forms and more. | Free and paid plans |
-| https://crt.sh | Get an overview and information of assigned TLS certificates. | Free |
+| https://crt.sh | Find The Real Vulnerabilities In The Development Phase Automatically.| Free & Paid |
```

Comment:
Checking on the crt.sh page I can't see any paid options, why was this change made? Can you provide a link to the paid option?

Reply:
This was not submitted by me